modules/repository-datastax/src/main/resources/repository.properties

```diff
@@ -19,3 +19,4 @@ export.pool.size:5
 default.time.format:${default.time.format}
 cassandra.mutagen.root.resource.path:${cassandra.mutagen.root.resource.path}
 mysql.mutagen.root.resource.path:${mysql.mutagen.root.resource.path}
+bucket.count.aggregation.interval:60
```

Comment:
@mooxter here value for bucket.count.aggregation.interval can be taken from pom.xml - similar to other configuration keys...